### DISCUSSION TITLE
没qq群貌似，要不开个￥5入的群？

### DISCUSSION THREAD

**Discussion by adoin** [vbenjs/vue-vben-admin#2296] [Ideas]
没qq群貌似，要不开个￥5入的群，讨论讨论啥的？

**Comment by qiekenol** [comment]
你来开

**Comment by adoin** [comment]
没号召力啊。。



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年11月11日(星期五) 下午5:07
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [vbenjs/vue-vben-admin] 没qq群貌似，要不开个￥5入的群？ (Discussion #2296)





 
你来开
